src/PageLayout/Notification/NotificationDetailsController.js

```diff
@@ -0,0 +1,217 @@
+import React, { Component } from 'react';
+import './Popup.css';
+import RecurringRequestController from './Recurring/RecurringRequestController';
+import close from '../../icons/cross-out.svg';
+import { NotificationType, NotificationCategory } from '../../Enums';
+import RecurringRequestCancelled from './Recurring/RecurringRequestCanceled';
+import { NotificationMap } from './NotificationMap';
+import RecurringRequestConfirmed from './Recurring/RecurringRequestConfirmed';
+
+class NotificationDetailsController extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            details: {
```

Comment:
i think you can just do `details: {}` here

Reply:
I get an error when I do that. Can we leave it as is since backend will take it over soon anyways?

Reply:
I get an error even when I put the setState outside the if. Can we leave it as is since backend will take over soon anyways?

Reply:
ya sure! Add a TODO comment here as well then?